.github/workflows/tz_update_check.yml

```diff
@@ -0,0 +1,43 @@
+name: Check for new IANA release
+
+on:
+  schedule:
+    - cron: "0 0 * * 0"
+  workflow_dispatch:
```

Comment:
not sure how familiar you are with GHA, but this allows you to trigger this action manually in the "Actions" tab